include/OsqpEigen/Data.tpp

```diff
@@ -7,26 +7,28 @@
 
 #include <iostream>
 
+#include "Debug.hpp"
```

Comment:
Please, include the header as for the others, i.e. ``#include <OsqpEigen/Debug.hpp>``.